.circleci/config.yml

```diff
@@ -20,9 +20,13 @@ commands:
   verify-slim-dependencies:
     steps:
       - run:
+          name: Set TAG to current sfdx version
           command: |
-            SFDX_CLI_VERSION="$(docker run -it --rm $DOCKER_IMAGE:$TAG bash -c 'sfdx --version')"
-            JAVA_VERSION="$(docker run -it --rm $DOCKER_IMAGE:$TAG bash -c 'java --version | head -n 1')"
+            echo "export TAG=$(jq -r .version ./package.json)-slim" >> $BASH_ENV
```

Comment:
We need the current sfdx version to be able to fetch the docker image right after we publish it, then we append `-slim` to get the right docker tag, same with `full`.

Reply:
some of the jobs have needed to have `      - jq/install` as a step.  Were you able to try this in a Circle environment to see if it would work without it?

Reply:
Yes, for both `docker-publish-*` jobs we use the image `cimg/node:lts` provided by CircleCI which has jq preinstalled:

```
➜  ~ docker run -it cimg/node:lts bash
circleci@078412ea1342:~/project$ which jq
/usr/bin/jq
circleci@078412ea1342:~/project$ jq --version
jq-1.6
```